include/nanogui/opengl.h

```diff
@@ -15,6 +15,10 @@
 
 #include <nanogui/common.h>
 
+// The Entypo PLUS and MINUS icons are 43 and 45 respectively, which needs to
+// be checked manually.
+#include <nanogui/entypo.h>
```

Comment:
Ooops.  Need to remove this, will happen in next commit on this branch.  This was the weird font encoding issue, was comparing to those two icon values explicitly in the helper functions.